src/views/basket.html

```diff
@@ -18,9 +18,10 @@ <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title
   </div>
   <div class="govuk-notification-banner__content">
     <p class="govuk-notification-banner__heading">
-      Your basket is full. You cannot add more than {{ basketLimit }} items to your order.
-      Remove some items before adding more.
+      Your basket is full. 
     </p>
+    <p class="govuk-body">You cannot add more than {{ basketLimit }} items to your order.
+      Remove some items before adding more.</p>
```

Comment:
This text does not seem to be exactly the same as in Phoebe's comment https://companieshouse.atlassian.net/browse/BI-12137?focusedCommentId=203241 ?